### PR TITLE
add Storybook stories for o-tabs

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Comment by notlee** [comment]
The demo text has too low contrast on the inverse demo. Instead of demo specific CSS I was thinking it might be possible to add a parameter for `color` as well as `background-color`
```
Inverse.parameters = {
	backgrounds: {
		default: 'slate',
	},
};
```

thoughts @chee ?

Screenshot 2022-01-21 at 12 44 59

File: components/o-tabs/stories/tabs.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	parameters: {
+		guidelines: {},
+		html: {},
+	},
+};
+
+const Story = args => {
+	useEffect(() => {
+		let tabs = javascript.init();
+		console.log({ tabs });
```
Comment (notlee) [inline components/o-tabs/stories/tabs.stories.tsx:23]:
```suggestion
```

**Comment by chee** [comment]
> The demo text has too low contrast on the inverse demo. Instead of demo specific CSS I was thinking it might be possible to add a parameter for `color` as well as `background-color`
> 
> ```
> Inverse.parameters = {
> 	backgrounds: {
> 		default: 'slate',
> 	},
> };
> ```
> 
> thoughts @chee ?
> 
> Screenshot 2022-01-21 at 12 44 59

we talked about this on Friday, and feel free to remove or ignore the `backgrounds` plugin and a similar plugin maybe just called `inverse` that can be false (body-text on page-background) or true ✅ (white on slate)

**Comment by chee** [comment]
> > The demo text has too low contrast on the inverse demo. Instead of demo specific CSS I was thinking it might be possible to add a parameter for `color` as well as `background-color`
> > ```
> > Inverse.parameters = {
> > 	backgrounds: {
> > 		default: 'slate',
> > 	},
> > };
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > thoughts @chee ?
> > Screenshot 2022-01-21 at 12 44 59
> 
> we talked about this on Friday, and feel free to remove or ignore the `backgrounds` plugin and a similar plugin maybe just called `inverse` that can be false (body-text on page-background) or true ✅ (white on slate)

after talking about this again, perhaps an `origami-themes` plugin that has options for `default` and `inverse`.

and i think maybe in `preview.js` we can replace the backgrounds config with `backgrounds: {disable: true }}` to disable the backgrounds button. that should only disable the backgrounds button, not the grid button :)

**Comment by notlee** [comment]
Blocked by: https://github.com/Financial-Times/origami/issues/589

**Comment by notlee** [comment]
@Financial-Times/origami-core this is ready for review! I'll merge in the [storybook addon](https://github.com/Financial-Times/origami/pull/595/), based on this branch, when this is approved

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**